clients/admin-ui/src/features/system/SystemFormTabs.tsx

```diff
@@ -182,6 +183,27 @@ const SystemFormTabs = ({
       ) : null,
       isDisabled: !activeSystem,
     },
+    {
+      label: "Data flow",
+      content: activeSystem ? (
+        <Box width={{ base: "100%", lg: "70%" }}>
+          <Box px={6} paddingBottom={2}>
+            <Text fontSize="md" lineHeight={6} fontWeight="bold">
+              Data flow
+            </Text>
```

Comment:
I think this may need some margin bottom?

hmm this doesn't look quite like the design to me.

| design | my instance |
|--------|------------|
|  <img width="402" alt="image" src="https://user-images.githubusercontent.com/24641006/230980676-6ec16ee3-58d6-4149-a903-695b6a8bd5c4.png"> |  <img width="685" alt="image" src="https://user-images.githubusercontent.com/24641006/230980591-621a5150-ccf7-41ab-b5d5-b74a2c922cb9.png"> |